index.html

```diff
@@ -22,17 +22,329 @@
         </nav>
     </nav>
      
-        <div class="container2">
-            <h1>Iptisam Souhel</h1>
+    <div class="container2">
+        <h1>Iptisam Souhel</h1>
+    </div>
+    <div class="container3">
+        <p>Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
+    If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
+    </div>
+    <div class="container4">
+        <div class="info">
+            <p>Start collaboration</p>
         </div>
-        <div class="container3">
-            <p>Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
-         If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
-        </div>
-        <div class="container4">
-            <div class="info">
-                <p>Start collaboration</p>
-            </div>
+    </div>
+    
+    <nav class="container5">
+        <ul class="images">
+            <li class="shape3"><img src="img/Shape1.png" alt="shape"></li>
+            <li class="shape4"><img src="img/Shape2.png" alt="shape"></li>
+            <li class="shape5"><img src="img/Shape3.png" alt="shape"></li>
+            <li class="shape6"><img src="img/Shape4.png" alt="shape"></li>
+            <li class="shape7"><img src="img/Shape5.png" alt="shape"></li>
+            <li class="shape8"><img src="img/Shape7.png" alt="shape"></li>
+            <li class="shape9"><img src="img/Shape8.png" alt="shape"></li>
+            <li class="shape10"><img src="img/Shape9.png" alt="shape"></li>
+            <li class="shape11"><img src="img/Shape10.png" alt="shape"></li>
+            <li class="shape12"><img src="img/Shape11.png" alt="shape"></li>
+            <li class="shape13"><img src="img/Shape12.png" alt="shape"></li>
+            <li class="shape14"><img src="img/Shape13.png" alt="shape"></li>
+            <li class="shape15"><img src="img/Shape14.png" alt="shape"></li>
+            <li class="shape16"><img src="img/Shape15.png" alt="shape"></li>
+            <li class="shape17"><img src="img/Shape.png" alt="shape"></li>
+            
+        </ul>
+    </nav>
+    <div class="container6">
+        <h2>Projects</h2>
+    </div>
+    
+    <nav class="container7">
```

Comment:
- [ ] Kindly wrap the section of cards in a section tag and remove all nav tags in cards this will make your website more semantic 👍🏿 
- [ ] Please add `display:grid` to the main container of this section 👍🏿